File: AndroidNetworking/Demo2/app/src/main/java/com/example/demo2/MainActivity.java
package com.example.demo2;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.widget.TextView;

public class MainActivity extends AppCompatActivity {

    TextView tv;
    //demo http get Request
    String get = "https://jsonplaceholder.typicode.com/posts/1";

    //demo http Post Request
    String post ="https://jsonplaceholder.typicode.com/posts";
//    param
//    body: JSON.stringify({
//        title: 'foo',
//                body: 'bar',
//                userId: 1,
//    }),
//    headers: {
//        'Content-type': 'application/json; charset=UTF-8',
//    },
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        tv = findViewById(R.id.tvKQ);

        findViewById(R.id.btnGet).setOnClickListener( view -> {
            Httpreques loader = new Httpreques(tv);
            loader.execute(get);
        });

        findViewById(R.id.btnPost).setOnClickListener( view -> {
            Httpreques loader = new Httpreques(tv);
            loader.execute(post);
        });

    }
}

File: MyApplication/app/src/main/java/com/example/myapplication/MainActivity.kt
package com.example.myapplication

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        val runnable = Runnable {
            val x = 1000
            for (i in 0..-1) {
            }
        }
        runnable.run()
    }
}